README.md

```diff
@@ -46,20 +48,19 @@ can do this by running the following commands:
     $ sudo sh -c "echo /usr/local/lib  >> /etc/ld.so.conf.d/local.conf"
     $ ldconfig
 
-## From the Git Repository
+## From a GitHub "Source Code" Archive / Git Repo Clone (Achtung!)
 
-To install from Git, you will need automake, autoconf, and libtool installed
-in addition to make and a compiler.
+**NOTE:** These instructions are for installation from the GitHub "Source Code" archives also available on the [Releases](https://github.com/maxmind/libmaxminddb/releases) page (e.g. `X.Y.Z.zip` or `X.Y.Z.tar.gz`), as well as installation directly from a clone of the [Git repo](https://github.com/maxmind/libmaxminddb). Installation from these sources are possible, but will present challenges to users not comfortable with manual dependency resolution.
 
-Our public git repository is hosted on GitHub at
-https://github.com/maxmind/libmaxminddb
+To install from a source code archive, you will need `automake`, `autoconf`, and `libtool` installed
```

Comment:
I think you could just remove "To install from a source code archive." It doesn't add anything and the release tarballs are "source code archives" too.